### DISCUSSION TITLE
Self-hosted BB Docker container affected by regular updates?

### DISCUSSION THREAD

**Discussion by dpedwards** [Budibase/budibase#5933] [General]
Hello dear BB dev team,
we are currently using our BB self-hosted version in a Docker container on AWS and wonder if our self-hosted version is also automatically affected by the latest Budibase updates. 

If so, how can we prevent updates from being applied automatically?

**Comment by shogunpurple** [comment]
Hi @dpedwards - could you show your `docker-compose.yaml`? 

Auto updates should be off by default on the `watchtower-service` container. It's possible you are using an old version of the `docker-compose.yaml` that has since been updated.

**Comment by dpedwards** [comment]
Hello, thanks for the tip. This thread can be closed now.
